apps/dashboard/pages/identity-portal/attendee-event-scan/index.tsx

```diff
@@ -181,10 +173,28 @@ export function Index() {
   }, [eventId]);
 
   useEffect(() => {
-    const timer = setInterval(fetchData, 5000);
-
-    return () => clearInterval(timer);
-  }, [attendees]);
+    if (eventId != null) {
+      const channel = supabase
+        .getClient()
+        .channel('db-changes')
+        .on(
+          'postgres_changes',
+          {
+            event: 'INSERT',
+            schema: 'public',
+            table: 'event_log',
+            filter: `event_id=eq.${eventId}`,
+          },
+          (payload) =>
+            updateData(payload.new.user_id, payload.new.check_in_time)
+        )
+        .subscribe();
+
+      return () => {
+        supabase.getClient().removeChannel(channel);
+      };
+    }
+  }, [eventId]);
```

Comment:
nit: add comment(s) to describe what this does

Reply:
Done!